tests/cli/test_cli_get_sample.py

```diff
@@ -0,0 +1,308 @@
+"""This script tests the cli methods to get samples in status-db"""
+from datetime import datetime
+
+from cg.store import Store
+
+
+def test_get_sample_bad_sample(invoke_cli, disk_store: Store):
+    """Test to get a sample using a non-existing sample-id """
+    # GIVEN an empty database
+
+    # WHEN getting a sample
+    db_uri = disk_store.uri
+    name = 'dummy_name'
+    result = invoke_cli(['--database', db_uri, 'get', 'sample', name])
+
+    # THEN then it should complain on missing sample id instead of getting a sample
+    assert result.exit_code == 0
```

Comment:
This can't be right? exit code should be non zero, right?
```suggestion
    assert result.exit_code == 1
```

Reply:
it will not fail since the API accepts multiple samples